ui/raidboss/data/03-hw/raid/a7s.js

```diff
@@ -175,4 +175,118 @@
       },
     },
   ],
+  timelineReplace: [
+    {
+      'locale': 'de',
+      'missingTranslations': true,
```

Comment:
Are there missing translations here? Maybe `find_missing_translations` should warn about any of these too.

Reply:
i thought i checked all the flags and removed them x.x

Reply:
ohh now i know, 
i left it in just for a7s, as it contains old log lines e.g. `suffers the effect of Pyretic` or even an empty elmenet `no translation for replaceText ''`

Reply:
Oh yeah, I marked a7s as `timelineNeedsFixing: true` and so those won't show up in `find_missing_timelines`.  I forgot that this is an old one.